docs/site/Deploying_to_ibm_cloud_kubernetes.md

```diff
@@ -0,0 +1,707 @@
+Try http://127.0.0.1:8080/ping
+````
+
+### Create a service
+
+Expose your deployment with a service named : lb4-simple-web-app-service
+
+````
+kubectl expose deployment/lb4-simple-web-app-deployment --type=NodePort --port=8080 --name=lb4-simple-web-app-service --target-port=8080
+
+````
+** NOTE : Notice the port numbers we are specifying here. They match the port numbers we used in the loopback4 application.
+
+
+### Obtain the NodePort of your service
+
+Let's determine the NodePort of your service.
+
+````
+kubectl describe service lb4-simple-web-app-service
+````
+
+You will see output like this:
+
+````
+Name:                     lb4-simple-web-app-service
+Namespace:                default
+Labels:                   run=lb4-simple-web-app-deployment
+Annotations:              <none>
+Selector:                 run=lb4-simple-web-app-deployment
+Type:                     NodePort
+IP:                       172.21.138.39
+Port:                     <unset>  8080/TCP
+TargetPort:               8080/TCP
+NodePort:                 <unset>  31701/TCP
+Endpoints:                172.30.29.115:8080
+Session Affinity:         None
+External Traffic Policy:  Cluster
+Events:                   <none>
+
+````
+
+In this case, the NodePort is 31701 .
+
+
+### Obtain the public IP address of the cluster
+
+Let's determine the public IP address of the cluster
+
+````
+ibmcloud ks workers dremondOne
+````
+
+You should see something like this"
+
+````
+OK
+ID                        Public IP       Private IP     Machine Type   State    Status
+kube-hou02-pa45e...6-w1   184.173.5.187   10.76.193.58   free           normal   Ready
+
+````
+
+In my case, the public IP is: 184.173.5.187
+
+
+So now we can formulate the url of our loopback4 application using those two pieces :
+
+http://184.173.5.187:31701
+
+
+
+Open your browser and attempt to access all these urls
+
+http://184.173.5.187:31701/
+
+http://184.173.5.187:31701/ping
+
+http://184.173.5.187:31701/explorer
+
+http://184.173.5.187:31701/openapi.json
+
+
+![](./imgs/lb4_k8s_ibm_cloud_app_website_on_cloud_k8s.png)
+
+
+
+## Step 17 - View your app in the Kubernetes Dashboard
+
+Let's go take a look at your application in the
+Kubernetes dashboard.
+
+Click the "Kubernetes Dashboard" button next to your cluster's name.
+
+![](./imgs/lb4_k8s_ibm_cloud_k8s_dashboard_1.png)
+
+
+Under "Workloads", select "Pods"
+
+![](./imgs/lb4_k8s_ibm_cloud_k8s_dashboard_2.png)
+
+
+Locate your application, and click on its name
+
+![](./imgs/lb4_k8s_ibm_cloud_k8s_dashboard_3.png)
+
+
+If you want to open a shell into the container in the pod, click on the
+[EXEC] button.
+
+If you want to view the logs of the container in the pod, click on the [LOGS] button.
+
+![](./imgs/lb4_k8s_ibm_cloud_k8s_dashboard_4.png)
+
+
+So there you have it! You have successfully deployed a Loopback4 application to Kubernetes on the IBM Cloud.
+
+
+@author : Dominique Emond
+@date   : Dec 14th, 2018
```

Comment:
We don't use this kind of signatures/metadata, please remove both `@author` and `@date`.

Reply:
understood